src/pages/verify/e-ids/german-personalausweis.mdx

```diff
@@ -0,0 +1,170 @@
+
+Please refer to the German [Act on Identity Cards and Electronic Identification](https://www.gesetze-im-internet.de/englisch_pauswg/englisch_pauswg.html) for more details about what you are allowed to do with the data from the Personalausweis.
+</Highlight>
+
+The data groups are bundled together into *profiles* which contain collections of data groups.
+Criipto has defined some standard profiles for various use cases, and request pricing is based on these profiles.
+
+The table below contains a description of the data groups and standard profiles.
+
+<table>
+    <thead>
+        <tr>
+            <th>Profile</th>
+            <th>Data groups</th>
+            <th>Intended use case</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>Pseudonym</td>
+            <td>Pseudonym</td>
+            <td>User login after registration</td>
+        </tr>
+        <tr>
+            <td>Profile 1</td>
+            <td>Pseudonym, First Name, Family Name, Religious Order / Artistic Name, Academic Title, Date of Birth, Place of Birth, Birth Name</td>
+            <td>User registration/one-time authentication</td>
+        </tr>
+        <tr>
+            <td>Profile 2</td>
+            <td>Everything in Profile 1, plus Nationality and Address (normal place of residence)</td>
+            <td>Know-your-customer (KYC)</td>
+        </tr>
+        <tr>
+            <td>Profile 3</td>
+            <td>Everything in Profile 2, plus Document Type, Issuing Country (always "D" for Germany), Date of Expiry and Residence Permit I</td>
+            <td>Extended KYC with document metadata</td>
+        </tr>
+    </tbody>
+</table>
+
+For applications configured to use a `dynamic` `scope` strategy, a profile can be selected on each request using the `scope` parameter prefixed with `odis:`, e.g. `odis:Profile1`.
+For applications configured to use a `static` `scope` strategy, the profile will always be `Pseudonym`.
+
+It is possible to define your own profiles if you have special requirements; if you would like to do so, please contact sales.
+
+### Transaction information
+You can add a reference text to display to the user while they scan the physical Personalausweis with the AusweisApp.
+The reference text consists of a _template_ with a single placeholder and a _dynamic parameter_ called `txinfo`.
+If a template is registered for the client, `txinfo` _must_ be passed as a `login_hint` on each request.
+The template may be up to 118 characters long and can contain any characters.
+The `txinfo` parameter can be up to 32 characters long and can only contain alphanumeric characters (in particular, it cannot contain spaces).
+
+### Test authorize requests
+You can use Criipto's test environment to test your integration before ordering your own client credentials (see below).
```

Comment:
Could we provide an example for tests using Criipto's environment? 
Maybe authorize URL examples? Or a link to authorize URL builder?

Reply:
Added a link to the authorize URL builder in 70999a7.